core-processor/src/ext.rs

```diff
@@ -869,54 +869,57 @@ impl EnvExt for Ext {
         self.error_explanation = Some(TerminationReason::GasAllowanceExceeded.into());
         TerminationReason::GasAllowanceExceeded.into()
     }
+
+    fn runtime_cost(&self, costs: RuntimeCosts) -> u64 {
+        costs.token(&self.context.host_fn_weights).weight()
```

Comment:
Could you elaborate the purpose of the method?

Reply:
It will be used in next PR. Sorry for that, but it's too hard to split one work into several PRs in a good manner